sunpy/database/tables.py

```diff
@@ -694,15 +689,17 @@ def entries_from_file(file, default_waveunit=None,
             # NOTE: the key DATE-END or DATE_END is not part of the official
             # FITS standard, but many FITS files use it in their header
             elif key in ('DATE-END', 'DATE_END'):
-                entry.observation_time_end = parse_time(
-                    value,
-                    _time_string_parse_format=time_string_parse_format
-                )
+                try:
+                    dt = parse_time(value).datetime
+                except ValueError:
+                    dt = Time.strptime(value, time_string_parse_format).datetime
```

Comment:
What happens if you get here and `time_string_parse_format=''`?

Reply:
`ValueError`

Reply:
:+1: